ui/apps/platform/src/Containers/MainPage/Header/Header.tsx

```diff
@@ -10,14 +10,26 @@ import {
 import { BarsIcon } from '@patternfly/react-icons';
 
 import BrandLogo from 'Components/PatternFly/BrandLogo';
+import Banners from '../Banners/Banners';
+import PublicConfigHeader from '../PublicConfig/PublicConfigHeader';
 import MastheadToolbar from './MastheadToolbar';
+import Notifications from './Notifications';
+
+// Style rule for Notifications, PublicConfigHeader, and Banners elements.
+import './Header.css';
 
 function Header(): ReactElement {
     // PageToggleButton assumes isManagedSidebar prop of Page element.
     // aria-label="primary" prop makes header element a unique landmark.
     return (
-        <Masthead className="ignore-react-onclickoutside theme-dark">
-            <MastheadToggle>
+        <Masthead
+            className="ignore-react-onclickoutside theme-dark"
+            inset={{ default: 'insetNone' }}
```

Comment:
Because banners need full page width:
* `insetNone` for `Masthead`
* `pf-v5-u-pl-lg` for `MastheadToggle`
* `pf-v5-u-pr-lg` for `MastheadContent`